src/components/Button.js

```diff
@@ -0,0 +1,5 @@
+import '../styles/Button.css';
+
+export default function Button({ content }) {
+  return <button>{content}</button>;
+}
```

Comment:
Button과 같은 core 컴포넌트를 만들 때에도, 해당 태그의 사용 방식과 유사하도록 구성하는 게 좋습니다.
버튼의 경우 `<button>버튼!</button>`과 같이 자식 요소를 활용하는데, 여기서도 그에 맞게 `<Button>버튼 컴포넌트!</Button>`과 같이 사용하려면 children을 활용하시면 됩니다.
`<Button content="버튼 컴포넌트!" />` 보다는 명확하죠?

```suggestion
export default function Button({ children }) {
  return <button>{children}</button>;
}
```